preupg/xmlgen/compose.py

```diff
@@ -331,20 +315,19 @@ def get_xml_tree():
         return target_tree
 
     @staticmethod
-    def run_compose(dir_name, content=None, generate_from_ini=True):
+    def run_compose(dir_name, generate_from_ini=True):
         target_tree = ComposeXML.get_xml_tree()
         settings.UPGRADE_PATH = dir_name
         if os.path.exists(os.path.join(dir_name, settings.file_list_rules)):
             os.unlink(os.path.join(dir_name, settings.file_list_rules))
-        group_xmls = ComposeXML.collect_group_xmls(dir_name, content, 0,
+(??)        group_xmls = ComposeXML.collect_group_xmls(dir_name, 0,
```

Comment:
Actually I most probably fixed it before you did the review. Maybe you had it opened for a few hours and then proceeded with the review :)